README.md

```diff
@@ -162,6 +162,8 @@ A tier within a tier group may identify data objects which are in violation by a
 
 Data objects which have been labeled via particular metadata, or within a specific collection, owned by a particular user, or belonging to a particular project may be identified through a custom query.  The default attribute **irods::storage_tiering::query** is used to hold this custom query.  To configure the custom query, attach the query to the root resource of the tier within the tier group.  This query will be used in place of the default time-based query for that tier.  For efficiency this example query checks for the existence in the root resource's list of leaves by resource ID.  Please note that any custom query must return DATA_NAME, COLL_NAME, USER_NAME, USER_ZONE, DATA_REPL_NUM in that order as it is a convention of this rule engine plugin.
 
+**Checking for resources in violating queries is required to prevent erroneous attempted data migrations for replicas on other resources which may be other tiers in the storage tiering group.** This can be done in the manner shown below (`DATA_RESC_ID in ('10068', '10069')`) or via resource hierarchy (e.g. `DATA_RESC_HIER like 'root_resc;%`), but the query must filter on resources to correctly identifying violating objects.
```

Comment:
`erroneous attempted data migrations` -> `erroneous data migrations`?

Reply:
I would rather keep the wording as is, or, if anything, change it to "erroneous scheduled data migrations". The data migrations are indeed erroneous, but the fact that they're being attempted/scheduled to begin with is what I'm trying to highlight here (given the asynchronous nature of the operations). Thoughts?

Reply:
That's fair. The decision to change or leave as is is yours.

Reply:
I removed the word "attempted".